code/Example/Drasil/DocumentLanguage.hs

```diff
@@ -245,8 +245,8 @@ mkSections si l = map doit l
 -- | Helper for creating the reference section and subsections
 mkRefSec :: SystemInformation -> RefSec -> Section
 mkRefSec _  (RefVerb s) = s
-mkRefSec si (RefProg c l) = section (titleize refmat) [c]
-  (map (mkSubRef si) l) "RefMat" "RefMat"
+mkRefSec si (RefProg c l) = section'' (titleize refmat) [c]
+  (map (mkSubRef si) l) "RefMat"
```

Comment:
I will be creating an issue that will ask to change all `section` constructor calls to either `section'` or `section''`, as it seems that that is a pattern when creating shortnames (i.e. condensing the title, or just the reference address, respectively). This may aid in tracking any patterns necessary to make the implementation of shortname cleaner.